doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Install-section.adoc

```diff
@@ -84,197 +85,377 @@ container. The Web container version of the WAR contains only the REST interface
 |===
+endif::[]
 
-[IMPORTANT]
-====
-If you are running both {KIE_SERVER} and {CENTRAL} you must configure KIE Server to use a different Data Source to {CENTRAL} using the  *org.kie.server.persistence.ds* property.
-{CENTRAL} uses a jBPM Executor Service that can conflict with {KIE_SERVER} if they share the same Data Source.
-====
+.System properties for executor properties
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.executor.interval`
+|Integer
+|`0`
+|The time between the moment the {PRODUCT} executor finishes a job and the moment it starts a new one, in a time unit specified in the `org.kie.executor.timeunit` property.
+
+|`org.kie.executor.timeunit`
+| https://docs.oracle.com/javase/8/docs/api/index.html?java/util/concurrent/TimeUnit.html[`java.util.concurrent.TimeUnit`] constant
+|`SECONDS`
+|The time unit in which the `org.kie.executor.interval` property is specified.
+
+|`org.kie.executor.pool.size`
+|Integer
+|`1`
+|The number of threads used by the {PRODUCT} executor.
+
+|`org.kie.executor.retry.count`
+|Integer
+|`3`
+|The number of retries the {PRODUCT} executor attempts on a failed job.
+
+|`org.kie.executor.jms.queue`
+|String
+|`queue/KIE.SERVER.EXECUTOR`
+|Job executor JMS queue for {KIE_SERVER}.
+|===
+
+ifdef::PAM,JBPM[]
+.Human task system properties
+[cols="2,1a,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.jbpm.ht.callback`
+|`mvel`
+
+`ldap`
+
+`db`
+
+`jaas`
+
+`props`
+
+`custom`
+|`jaas`
+|A property that specifies the implementation of user group callback to be used:
+
+* `mvel`: Default; mostly used for testing.
+* `ldap`: LDAP; requires additional configuration in the `jbpm.usergroup.callback.properties` file.
+* `db`: Database; requires additional configuration in the `jbpm.usergroup.callback.properties` file.
+* `jaas`: JAAS; delegates to the container to fetch information about user data.
+* `props`: A simple property file; requires additional file that will keep all information (users and groups).
+* `custom`: A custom implementation; specify the fully qualified name of the class in the `org.jbpm.ht.custom.callback` property.
+
+|`org.jbpm.ht.custom.callback`
+|Fully qualified name
+|N/A
+|A custom implementation of the `UserGroupCallback` interface in case the `org.jbpm.ht.callback` property is set to `custom`.
+
+|`org.jbpm.task.cleanup.enabled`
+|`true`, `false`
+|`true`
+|Enables task cleanup job listener to remove tasks once the process instance is completed.
+
+|`org.jbpm.task.bam.enabled`
+|`true`, `false`
+|`true`
+|Enables task BAM module to store task related information.
+
+|`org.jbpm.ht.admin.user`
+|String
+|`Administrator`
+|User who can access all the tasks from {KIE_SERVER}.
+
+|`org.jbpm.ht.admin.group`
+|String
+|`Administrators`
+|In order to display all the tasks from {KIE_SERVER}, users must belong to the `Administrators` group.
+|===
+endif::[]
+
+.System properties for loading keystore
```

Comment:
All system properties for keystores don't contain any default value, please remove them.

Reply:
Ack